content/news/buzz/2025-curtin-capstone-semester-end-blog/contents.lr

```diff
@@ -0,0 +1,86 @@
+  
+We were first tasked with contributing to a first timer issue before moving on to other more advanced issues.
+
+### Briefcase Contributions
+
+- PR [#2198](https://github.com/beeware/briefcase/pull/2198): Add boolean question (feature)
+- PR [#2103](https://github.com/beeware/briefcase/pull/2203): Add XML content escaping filter to cookiecutter.py (feature)
+- PR [#2199](https://github.com/beeware/briefcase/pull/2199): Add catch exception when deleting JDK (bugfix)
+- PR [#2229](https://github.com/beeware/briefcase/pull/2229): Accept other changelog name and extension (feature)
+- PR [#2201](https://github.com/beeware/briefcase/pull/2201): Add ``--app`` option to briefcase build/package (feature)
+- PR [#2214](https://github.com/beeware/briefcase/pull/2214): Add ``--app`` option to briefcase create/update (feature)
+- PR [#2236](https://github.com/beeware/briefcase/pull/2236): Normalise contribution docs with Toga (documentation)
+- PR [#54](https://github.com/beeware/briefcase-windows-VisualStudio-template/pull/54): Use XML content escaping filter to Visual Studio template. (bugfix) (work in progress)
+
+### Toga Contributions
+
+- PR [#3259](https://github.com/beeware/toga/pull/3259): Add web screenshots (documentation)
+- PR [#3466](https://github.com/beeware/toga/pull/3466): Fix button click handling in Toga Web backend to correctly trigger event (bugfix)
+- PR [#3338](https://github.com/beeware/toga/pull/3338): DateInput (web widget)
+- PR [#3405](https://github.com/beeware/toga/pull/3405): TimeInput (web widget)
+- PR [#3362](https://github.com/beeware/toga/pull/3362): ScrollContainer (web widget)
+- PR [#3425](https://github.com/beeware/toga/pull/3425): Table (web widget) (work in progress)
+- PR [#3402](https://github.com/beeware/toga/pull/3402): Selection (web widget)
+- PR [#3527](https://github.com/beeware/toga/pull/3527): Slider (web widget)
+
+## What We're Working Towards
+
+Now that we’re more familiar with the BeeWare ecosystem, through both our initial PRs and widget implementations, and having concluded our Semester 1 research phase, we’re shifting our focus to larger, more structural improvements for the second half of the project.
+
+Our primary goals moving forward include:
+
+### Toga Web Testing
+
+Toga does not currently support testing for the web platform as the test suite and testbed app run multithreaded in the same process. The web platform is incompatible with this as WebAssembly does not support multithreading. To address this, we are working on developing a general approach that allows the test suite and testbed app to run in separate processes. This will include using proxy objects in place of Toga objects in the test suite that will ‘wrap’ the respective object in the testbed app and ‘forward’ commands to it in order to change its state and retrieve data from it
```

Comment:
As noted in a previous review - there's no need to 'quote' words like this; and if you did, you wouldn't use smart quotes:
```suggestion
Toga does not currently support testing for the web platform as the test suite and testbed app run multithreaded in the same process. The web platform is incompatible with this as WebAssembly does not support multithreading. To address this, we are working on developing a general approach that allows the test suite and testbed app to run in separate processes. This will include using proxy objects in place of Toga objects in the test suite that will wrap the respective object in the testbed app, forwarding commands to it in order to change its state and retrieve data.
```